source/CaptureOdds.h

```diff
@@ -14,8 +14,10 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 #define CAPTURE_ODDS_H_
 
 #include <vector>
+#include "Government.h"
```

Comment:
This should be the new line 16, and the std include should then be line 18, to follow the style guide.